### DISCUSSION TITLE
Create `@js` blade directive

### DISCUSSION THREAD

**Discussion by calebdw** [laravel/framework#46609] [Ideas] [answered]
The current practice for bringing PHP data from the backend to the frontend is using `Js::from`, however, Livewire has an [`@js` directive](https://laravel-livewire.com/docs/2.x/inline-scripts#using-js-directive) which is much more verbose (and doesn't necessitate mustache syntax).

Would it be possible to create a `@js` directive that wraps around `Js::from`?

Current:
```blade
    <script>
        var app = {{ Illuminate\Support\Js::from($array) }};
        {{-- or --}}
        var app = {{ Js::from($array) }};
    </script>
```

Desired:
```blade
    <script>
        var app = @js($array);
    </script>
```

Thanks!

**Comment by browner12** [comment, marked as answer]
This exists already (but isn't currently documented).

https://github.com/laravel/framework/blob/10.x/src/Illuminate/View/Compilers/Concerns/CompilesJs.php

**Reply by calebdw** [reply]
Awesome! Can we document the feature?

**Reply by browner12** [reply]
go right ahead.

**Reply by calebdw** [reply]
Documented in [#8687](https://github.com/laravel/docs/pull/8687)

**Comment by calebdw** [comment]
PR was denied so nothing to do here
